packages/oslo-generator-shacl-template/test/util.ts

```diff
@@ -0,0 +1,14 @@
+import type * as RDF from '@rdfjs/types';
+import rdfParser from 'rdf-parse';
+
```

Comment:
I'm gonna be super nitpicky here, but `util.ts` as filename feels a bit off to me 😆 . Nothing really buggy about it ofcourse!

Reply:
I use the `utils.ts` file to store util functions for the package. I'll change it to `test-utils.ts` for now. Let's discuss this later if you still want to rename it!